src/server/game.ts

```diff
@@ -108,6 +110,24 @@ export function newGame (whitePlayer: Player, initState?: Board, initDisc?: Disc
   return notify(EVENT_GAME_CREATED, state(game.id, whitePlayer))
 }
 
+export function reMatch (gameId: string, action: string) : Result {
+  if(action === 'accept'){
+    let game = {
+      id: gameId,
+      whitePlayer: games[gameId].whitePlayer,
+      blackPlayer: games[gameId].blackPlayer,
+      status: gameStatus.STATUS_PENDING,
+      turn: 
+      // initDisc || previous winner begins?
```

Comment:
White and Black players should switch side on rematch.

Reply:
Fixed. Added logging of who begins at every new game and rematch as well.